app/controllers/api/categories/items_controller.rb

```diff
@@ -2,10 +2,10 @@ module Api
   module Categories
     class ItemsController < ApiController
       def index
-        # chosen_item = Item.where(category: params[:category_id]) works aswell
         chosen_items = Item.all.select { |i| i.category.downcase == params[:category_id] }
+        serializer = ApiCategoriesSerializer.new
```

Comment:
The name of the serialiser does not match the context 🤔 
If you consider this, you are parsing Item objects, so the serialiser should be `ItemsSerializer`. If the name is already reserved / there is a difference serialiser with this name, put it into a module so that it would be `Categories::ItemsSerializer`.

Reply:
It's actually better to have it a module, so do this anyway.